Enable graphic console on isapc (addressed by upstream r5026)
Lubomir Rintel <lkundrak@v3.sk>

--- qemu-0.9.1/hw/cirrus_vga.c.orig	2008-01-06 20:38:42.000000000 +0100
+++ qemu-0.9.1/hw/cirrus_vga.c	2008-12-03 11:03:21.000000000 +0100
@@ -3197,6 +3197,7 @@ void isa_cirrus_vga_init(DisplayState *d
     vga_common_init((VGAState *)s,
                     ds, vga_ram_base, vga_ram_offset, vga_ram_size);
     cirrus_init_common(s, CIRRUS_ID_CLGD5430, 0);
+    graphic_console_init(s->ds, s->update, s->invalidate, s->screen_dump, s);
     /* XXX ISA-LFB support */
 }
 
